include/pops/model.hpp

```diff
@@ -141,7 +141,7 @@ class Model
                 dispersers,
                 infected,
                 config_.weather,
-                weather_coefficients[weather_step],
```

Comment:
if possible weather_step could be removed from line 73.

Reply:
Huh, I relied on compiler warnings, but forgot I did not enable them for CMake. Thanks for catching that.